docker-compose.build.yml

```diff
@@ -10,8 +10,8 @@ services:
     ## Slight warning that it does take significantly more time to build than
     ## the other services though.
     deploy:
-      replicas: 0
-    #build: wharf-web
+      replicas: 1
+    build: wharf-web
```

Comment:
What is the motivation for this change?

Reply:
Oops, I forgot this was a draft and that it's probably part of testing?

Reply:
Reverted.